teams/hubturbo.md

```diff
@@ -30,7 +30,7 @@ permalink: /teams/hubturbo/
 
  - [@xinan](https://github.com/xinan)
    ([Liu Xinan](http://xinan.io))
-   - Technology Expertise:
+   - Technology Expertise: Performance Optimization
```

Comment:
Okay. I will make it more specific by changing it to "UI Responsiveness Optimization".